### PR TITLE
feat: argocd-notifications-controller log level and log format be defined with environment variables

### PR DESCRIPTION
**Pull request by noaabarki**
This is a simple fix that does the following: 
Allowing `argocd-notifications-controller` log level and log format be defined with environment variables. 
This PR solved [this](https://github.com/argoproj/argo-cd/issues/13039) issue. 

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [ ] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md#legal)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)). 
* [ ] My new feature complies with the [feature status](https://github.com/argoproj/argoproj/blob/master/community/feature-status.md) guidelines.
* [ ] I have added a brief description of why this PR is necessary and/or what this PR solves.

Please see [Contribution FAQs](https://argo-cd.readthedocs.io/en/latest/developer-guide/faq/) if you have questions about your pull-request.

### CODE REVIEW

File: manifests/install.yaml [hunk 1]
```diff
@@ -17240,7 +17253,7 @@ spec:
           - key: tls.key
             path: tls.key
           - key: ca.crt
-            path: ca.crt
+            path: ca.crtrgocd-notifications-controller
```
Comment (morey-tech) [inline manifests/install.yaml:17256]: I think this is an accidental addition.
```suggestion
            path: ca.crt
```

**Review by morey-tech — CHANGES_REQUESTED** [review]
Thank you for adding this! I've gotten the question of how to set the notifications controller log level many times.

File: manifests/install.yaml (hunk 1 above)
Comment (noaabarki) [reply manifests/install.yaml:17256]: Oh, thank you! Completely my bad, fixing it right now.

**Review by crenshaw-dev — APPROVED** [review]
Thanks, @noaabarki!

**Comment by crenshaw-dev** [comment]
@noaabarki can you sign off your commits? Let me know if you need any help, the instructions aren't always intuitive.

**Comment by noaabarki** [comment]
> @noaabarki can you sign off your commits? Let me know if you need any help, the instructions aren't always intuitive.

Thank you very much @crenshaw-dev!! 🙏🏼 That would be great, I'm not sure how to go about it, so I'd really appreciate your help.

**Comment by blakepettersson** [comment]
@noaabarki you should be able do it like this (assuming your Git version >= 2.13)

```
git rebase --signoff HEAD~8
```

**Comment by noaabarki** [comment]
> git rebase --signoff HEAD~8

Thank you @blakepettersson. I'm worried it will sign-off other contributor's commits.

**Review by drpaneas — APPROVED** [review]
/lgtm

**Comment by drpaneas** [comment]
@noaabarki you can always create a new PR with one commit and we can close this one, if rebasing is not comfortable.

**Comment by noaabarki** [comment]
> @noaabarki you can always create a new PR with one commit and we can close this one, if rebasing is not comfortable.

Hi @drpaneas! I created a new PR with one commit https://github.com/argoproj/argo-cd/pull/13605. Let me know what you think :)

**Comment by thiagoscodelerae** [comment]
When are you planning to release this feature?

**Pull request closed without merging**